packages/shylu/shylu_dd/frosch/src/Tools/FROSch_Output.h

```diff
@@ -55,11 +55,11 @@
         const bool throw_exception = !(COND); \
         if(throw_exception) { \
             Teuchos::TestForException_incrThrowNumber(); \
-            const int throwNumber = throwNumber; \
+            const int throwNumber = Teuchos::TestForException_getThrowNumber(); \
```

Comment:
This was unrelated to fastilu, but was generating tons of warnings, so I went ahead and fixed it (I think this is the right fix).